src/System.Runtime.Serialization.Formatters/ref/project.json

```diff
@@ -1,5 +1,8 @@
 {
   "dependencies": {
+    "System.Collections.NonGeneric": "4.0.0",
```

Comment:
(I don't actually mean this file, but it's hard to talk about files not in the change)

Have we already bumped the contract and framework version for this package?